sdk/ai/ai-projects/src/agents/index.ts

```diff
@@ -308,8 +308,8 @@ function getAgents(context: Client): AgentsOperations {
     deleteThread: (threadId: string, requestParams?: OptionalRequestParameters) =>
       deleteThread(context, threadId, requestParams),
 
-    createRun: (threadId: string, assistantId: string, options?: Omit<CreateRunOptions, "assistant_id">, requestParams?: OptionalRequestParameters) =>
-      createRun(context, threadId, { ...requestParams, body: { ...options, assistant_id: assistantId, } }),
+    createRun: (threadId: string, assistantId: string, options?: CreateRunOptionalParams) =>
+      createRun(context, threadId, assistantId, options || {assistantId}),
```

Comment:
nit: `??` means "only if the first operand is null or undefined" which is slightly better for cases where it might coerce to be falsy but still is valid. It shouldn't matter here because options is an object, but it's a good habit to get into

```suggestion
      createRun(context, threadId, assistantId, options ?? {assistantId}),
```